### PR TITLE
ADEN-7152 Track only first pos value

### PR DESCRIPTION
**Pull request by v3rth**
## Links

* https://wikia-inc.atlassian.net/browse/ADEN-7152

### CODE REVIEW

File: app/modules/ads/tracking/slot-tracker.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 const trackingRouteName = 'special/adengadinfo';
 
+function getPosParameter({pos = ''}) {
```
Comment (nandy-andy) [inline app/modules/ads/tracking/slot-tracker.js:3]: We can unit test it! 🙌

File: app/modules/ads/tracking/slot-tracker.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+	return (Array.isArray(pos) ? pos : pos.split(',')).shift();
```
Comment (fraszczakszymon) [inline app/modules/ads/tracking/slot-tracker.js]: `shift` modifies original array. Can we get first element without changing reference?

Comment (v3rth) [reply app/modules/ads/tracking/slot-tracker.js]: You are right! thanks!

File: app/modules/ads/tracking/slot-tracker.js (hunk 1 above)
Comment (v3rth) [reply app/modules/ads/tracking/slot-tracker.js:3]: Added :heavy_check_mark:

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**